nav-app/src/app/datatable/data-table.component.html

```diff
@@ -305,6 +305,16 @@
                  </div>
              </div>
 
+            <!-- show or hide disabled techniques -->
+            <div *ngIf="configService.getFeature('toggle_subtechniques')" class="control-row-item">
+                <div class="control-row-button noselect"
+                     (click)="viewModel.showAllSubtechniques = !viewModel.showAllSubtechniques;"
+                     matTooltipPosition="below"
+                     matTooltip="show/hide sub-techniques">
+                     <img [src]="viewModel.showAllSubtechniques ? 'assets/icons/label_24px.svg' : 'assets/icons/label_off_24px.svg'" [alt]="viewModel.showAllSubtechniques ? 'hide sub-techniques' : 'show sub-techniques'"/>
+                 </div>
```

Comment:
This should probably be two different buttons. See my comment on the `showAllSubtechniques` field for more details, but we essentially need one button to set all `showSubtechniques` fields to `true` and one button to set them all to `false`.

A toggle like this _could_ work with that UX but it wouldn't be optimal -- if the user had one technique open and the rest closed, would clicking the toggle expand all or collapse all sub-techniques? The simpler UX is to simply have two buttons instead of a toggle.